cloudbridge/cloud/providers/aws/resources.py

```diff
@@ -559,29 +479,28 @@ def id(self):
 
     @property
     def name(self):
-        """
-        Get the snapshot name.
-
-        .. note:: an instance must have a (case sensitive) tag ``Name``
-        """
-        return self._snapshot.tags.get('Name')
+        for tag in self._snapshot.tags or list():
```

Comment:
This construct for getting the name keeps repeating. Should it be extracted into a private helper method?

Reply:
Good point. I'd been planning to do that and forgot.

Reply:
Done